operator/api/v1alpha1/keptntaskdefinition_types.go

```diff
@@ -35,6 +35,8 @@ type FunctionSpec struct {
 	ConfigMapReference ConfigMapReference `json:"configMapRef,omitempty"`
 	Parameters         TaskParameters     `json:"parameters,omitempty"`
 	SecureParameters   SecureParameters   `json:"secureParameters,omitempty"`
+	Retries            Retries            `json:"retries,omitempty"`
+	Timeouts           Timeouts           `json:"timeouts,omitempty"`
```

Comment:
Let's keep it singular since it is a single value 
```suggestion
	Timeout           Timeout           `json:"timeouts,omitempty"`
```

Reply:
Should this be a Duration instead of an int?